libnetwork/osl/interface_unsupported.go

```diff
@@ -2,4 +2,59 @@
 
 package osl
 
-type nwIface struct{}
+import (
+	"fmt"
+	"net"
+
+	"github.com/docker/docker/libnetwork/types"
+)
+
+type Interface struct{}
+
+func (i *Interface) SrcName() string {
+	return ""
+}
```

Comment:
Need to remove all of this, because it looks like this is only needed because `libnetwork/osl` is imported in some windows files, but not actually doing anything.